.github/workflows/CI.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           - OptimizationPolyalgorithms
         version:
           - '1'
-          - '1.6'
+          - '1.8'
```

Comment:
@ChrisRackauckas are you okay with dropping 1.6 testing?

Reply:
No, we still support LTS. Why would it be dropped?

Reply:
I thought there was some movement towards dropping 1.6 from CI.. doesn't really matter here so no worries